code/common/execas.q

```diff
@@ -1,4 +1,4 @@
 \d .pm
 
 execas:{[f;u]
- $[`.pm.requ ~ key `.pm.requ;.pm.requ[u;f]; value f]}
+ $[`.pm.requ ~ key `.pm.requ;.pm.requ[u;f]; valp f]}
```

Comment:
This now requires permissions.q to be loaded. Can you add a check to see if valp exists and only use it if it does